### ISSUE TITLE
Question about banner

### ISSUE THREAD

**Issue by lindsaycarr** [USGS-VIZLAB/water-use-15#307]
Reviewer: 

> This banner doesn’t sit right with me.  It seems that the USGS logo should be separate from the title of the image.  And if the logo appears at the top of the page, why is it also floating below in the image?

![image](https://user-images.githubusercontent.com/13220910/39581248-5ca49aa6-4ea8-11e8-9b2b-c26989d76fab.png)

The answer to the floating one is that this can be embedded, so we want our logo to come along with it.  This is the second comment we've had about the title next to the logo in the banner. I don't think we are violating anything in vis id: https://www.usgs.gov/information-policies-and-instructions/usgs-visual-identity-system.

**Comment by jread-usgs** [comment]
We have lost this battle in the past, hence changing the banner https://owi.usgs.gov/vizlab/climate-change-walleye-bass/ to be Shoemaker-eligible. We should probably have a header solution that doesn't get dinged by different interpretations of vizID policy. 

Another reason the watermark logo appears on the map is that people screenshot it and this way USGS doesn't lose attribution.

**Comment by lindsaycarr** [comment]
Right, yes - good point. So what's your thought on the banner? Should we work on fixing it so the title is below the logo (at least on desktop)?

**Comment by jread-usgs** [comment]
We spent a lot of time discussing this banner: https://marine.usgs.gov/coastalchangehazardsportal/ which is similar to what we do in vizlab. The reason I argued that app doesn't fit into the vizID rules that are coming up here is that it is considered a webapp, vs a viz in our case may look like an app but is static files. Not that that should matter much. 

I think we can improve what we have currently (e.g., the title seems neither vertically centered or to share the same baseline as the "USGS" and keep the similar collapsed/thin design for at least mobile/tablet, while experimenting w/ some ways of passing the visID rules on desktop. The way we did it on the climate-fish viz was a quick fix, and I think we can do better here if we want to pursue it.
